src/main/java/net/sf/jabref/model/entry/specialfields/SpecialField.java

```diff
@@ -0,0 +1,111 @@
+package net.sf.jabref.model.entry.specialfields;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Optional;
+
+import net.sf.jabref.model.entry.KeywordList;
+
+public enum SpecialField {
+
+    PRINTED("printed",
+        SpecialFieldValue.PRINTED
+    ),
+
+    PRIORITY("priority",
+        SpecialFieldValue.CLEAR_PRIORITY,
+        SpecialFieldValue.PRIORITY_HIGH,
+        SpecialFieldValue.PRIORITY_MEDIUM,
+        SpecialFieldValue.PRIORITY_LOW
+    ),
+
+    QUALITY("qualityassured",
+        SpecialFieldValue.QUALITY_ASSURED
+    ),
+
+    RANKING("ranking",
+        SpecialFieldValue.CLEAR_RANK,
+        SpecialFieldValue.RANK_1,
+        SpecialFieldValue.RANK_2,
+        SpecialFieldValue.RANK_3,
+        SpecialFieldValue.RANK_4,
+        SpecialFieldValue.RANK_5
+    ),
+
+    READ_STATUS("readstatus",
+        SpecialFieldValue.CLEAR_READ_STATUS,
+        SpecialFieldValue.READ,
+        SpecialFieldValue.SKIMMED
+    ),
+
+    RELEVANCE("relevance",
+        SpecialFieldValue.RELEVANT
```

Comment:
Why no CLEAR_RELEVANT? Maybe a comment should be added to the class itself how toggle values are distinguished from multi-values. Is it only because of the number of values given to the enum?